### PR TITLE
Corrections of 0.12 release notes

### PR DESCRIPTION
**Pull request by instagibbs**

### CODE REVIEW

File: doc/release-notes/release-notes-0.12.0.md [hunk 1]
```diff
@@ -166,12 +166,13 @@ creating transactions that would be replaceable under BIP 125.
 RPC: Random-cookie RPC authentication
 -------------------------------------
 
-When no `-rpcpassword` is specified, the daemon now uses a special 'cookie'
-file for authentication. This file is generated with random content when the
+The daemon now uses a special 'cookie' file for authentication.
```
Comment (laanwj) [inline doc/release-notes/release-notes-0.12.0.md]: I'm not sure this is more clear. Cookie authentication is only used when no `-rpcpassword` is specified, it's not suddenly used in previous setups.

Comment (instagibbs) [inline doc/release-notes/release-notes-0.12.0.md]: ok I derped. Someone else had tested this(incorrectly?) on IRC and I remembered wrong. It's rpcauth which is tried after cookie auth.

**Comment by Michagogo** [comment]
This is still inaccurate. Rescans don't work. This is true whether they're manually initiated via the rescan flag or as part of one of those RPCs. I realize the intent is to make it clear that those commands are still possible sans rescan, but it goes too far and now doesn't indicate that a standalone rescan won't work.

**Comment by jonasschnelli** [comment]
IMO the current release notes (without this PR) is okay. Unless we have a rescan-down-to-height feature (which we don't have in 0.12), rescans don't work with pruning (current rescans always start at genesis).
Tend to NACK.

**Comment by laanwj** [comment]
I don't think this change is correct either. `-rescan` doesn't work with pruning, period. RPC calls such as import will work without rescan, though.

I'm sure that could be worded better, though.

But there's a limit on how much time we should spend on polishing release notes on a release that's already "history". Let's try to have the next releases's release notes ready in time for the next release :-)

**Comment by instagibbs** [comment]
Agreed

**Pull request closed without merging**